edb/edgeql/parser/grammar/expressions.py

```diff
@@ -947,8 +946,8 @@ def reduce_SCONST(self, str_tok):
         quote = match.group('Q')
         val = match.group('body')
 
-        # handle line continuations
-        val = re.sub(r'\\\n', '', val)
+        # collapse the whitespace after a line continuation
+        val = lexutils.to_sql_text(val)
```

Comment:
At this point we're very far from any SQL, so I'd choose a different name here, like "collapse_newline_whitespace", which is also more informative as to what's going on.

Reply:
Yeah, I originally thought it was going to be closer to SQL. I renamed the function to `collapse_newline_whitespace`.